client/blocks/inline-help/inline-help-search-card.jsx

```diff
@@ -33,13 +33,22 @@ class InlineHelpSearchCard extends Component {
 		openResult: PropTypes.func.isRequired,
 		translate: PropTypes.func,
 		query: PropTypes.string,
+		placeholder: PropTypes.string,
+		location: PropTypes.string,
```

Comment:
This isn't a blocker, but I think we should be careful about adding any more props to this component. The API surface is expanding a lot. If it increases further later on we might like to consider inversion of control via hooks or render props or something to allow the consumer to decide on the specifics rather than making the component do everything.

This will be ok for now though.